kubernetes/config-maps/nginx-storage/default.conf.j2

```diff
@@ -0,0 +1,94 @@
+server {
+    listen 80;
+
+    gzip                     on;
+    gzip_min_length          1000;
+    gzip_proxied             expired no-cache no-store private auth;
+    gzip_types               text/plain text/css text/javascript application/xml;
+    client_max_body_size     {{ max_upload_size }};
+
+
+    location = /upload/ {
+        access_by_lua_block {
+            -- We're using nginx-big-upload module which supports only "application/x-www-form-urlencoded".
+            -- Return an error if the content type is different.
+            if ngx.req.get_headers()["Content-Type"] ~= "application/x-www-form-urlencoded" then
+                ngx.status = 415
+                ngx.say('Unsupported content type. The only supported type is application/x-www-form-urlencoded.')
```

Comment:
You're returning a text string here. It should be a JSON message, with appropriate `Content-Type`.

Reply:
Done.